Gemfile

```diff
@@ -2,9 +2,6 @@ source("https://rubygems.org")
 
 gem "xcode-install", ">= 2.2.1" # needed for running xcode-install related tests
 
-gem "danger", ">= 4.2.1", "< 5.0.0"
-gem "danger-junit", ">= 0.7.3", "< 1.0.0"
```

Comment:
Can we keep the `danger` and `danger-junit` in the `Gemfile`? They are used in CI and not for development so I _think_ this is still the correct place for them? 😇

Reply:
You got it!